### ISSUE TITLE
Error in writing mcmc file, Win version

### ISSUE THREAD

**Issue by nylander** [NBISweden/MrBayes#63]
Transferred from https://sourceforge.net/p/mrbayes/bugs/1655/

Created: 2018-01-03
Creator: Milos Macholan

I have downloaded and installed MrBayes 3.2.6 in C (x64-based processor, Windows 10 OS) and tried to analyse the "primates.nex" using the default values. After entering the "mcmc" command the following error message appeared:
Could not open file "primates.nex.mcmc"
Error preparing print files
Error in command "Mcmc"
Can you tell me what´s wrong? The older 3.1 version works without problems. Thank you

**Comment by kusalananda** [comment]
This one is a bit difficult to debug without knowing exactly what the run looked like. I can not reproduce this on OpenBSD using `mb -i primates.nex` and `mcmc` in the interactive session.

**Comment by nylander** [comment]
More details regarding this behavior now received (below). This seems to affect (at least) the compiled windows version for 3.2.6 on Win 10. This is a "won't fix" until new windows binaries for 3.2.7 can be tested.

"I am using Windows 10, installed MrBayes 3.2.6. and followed the manual MrBayes 3.2.:

```
execute primates.nex
lset nst=6
lset rates=invgamma
mcmcp diagnfreq=1000
mcmcp samplefreq=100
mcmcp printfreq=100
mcmc




                            MrBayes v3.2.6 x86

                      (Bayesian Analysis of Phylogeny)

              Distributed under the GNU General Public License


               Type "help" or "help <command>" for information
                     on the commands that are available.

                   Type "about" for authorship and general
                       information about the program.


MrBayes > exe primates.nex

   Executing file "primates.nex"
   DOS line termination
   Longest line length = 920
   Parsing file
   Expecting NEXUS formatted file
   Reading data block
      Allocated taxon set
      Allocated matrix
      Defining new matrix with 12 taxa and 898 characters
      Data is Dna
      Data matrix is not interleaved
      Gaps coded as -
      Taxon  1 -> Tarsius_syrichta
      Taxon  2 -> Lemur_catta
      Taxon  3 -> Homo_sapiens
      Taxon  4 -> Pan
      Taxon  5 -> Gorilla
      Taxon  6 -> Pongo
      Taxon  7 -> Hylobates
      Taxon  8 -> Macaca_fuscata
      Taxon  9 -> M_mulatta
      Taxon 10 -> M_fascicularis
      Taxon 11 -> M_sylvanus
      Taxon 12 -> Saimiri_sciureus
      Successfully read matrix
      Setting default partition (does not divide up characters)
      Setting model defaults
      Seed (for generating default start values) = 1552911463
      Setting output file names to "primates.nex.run<i>.<p|t>"
   Exiting data block
   Reached end of file

MrBayes > lset nst=6

   Setting Nst to 6
   Successfully set likelihood model parameters

MrBayes > lset rates=invgamma

   Setting Rates to Invgamma
   Successfully set likelihood model parameters

MrBayes > mcmcp diagnfreq=1000

   Setting diagnosing frequency to 1000
   Successfully set chain parameters

MrBayes > mcmcp samplefreq=100

   Setting sample frequency to 100
   Successfully set chain parameters

MrBayes > mcmcp printfreq=100

   Setting print frequency to 100
   Successfully set chain parameters

MrBayes > mcmc

   Running Markov chain
   MCMC stamp = 6033037634
   Seed = 125270998
   Swapseed = 1552911463
   Model settings:

      Data not partitioned --
         Datatype  = DNA
         Nucmodel  = 4by4
         Nst       = 6
                     Substitution rates, expressed as proportions
                     of the rate sum, have a Dirichlet prior
                     (1.00,1.00,1.00,1.00,1.00,1.00)
         Covarion  = No
         # States  = 4
                     State frequencies have a Dirichlet prior
                     (1.00,1.00,1.00,1.00)
         Rates     = Invgamma
                     The distribution is approximated using 4 categories.
                     Likelihood summarized over all rate categories in each generation.
                     Shape parameter is exponentially
                     distributed with parameter (1.00).
                     Proportion of invariable sites is uniformly dist-
                     ributed on the interval (0.00,1.00).

   Active parameters:

      Parameters
      ---------------------
      Revmat              1
      Statefreq           2
      Shape               3
      Pinvar              4
      Ratemultiplier      5
      Topology            6
      Brlens              7
      ---------------------

      1 --  Parameter  = Revmat
            Type       = Rates of reversible rate matrix
            Prior      = Dirichlet(1.00,1.00,1.00,1.00,1.00,1.00)

      2 --  Parameter  = Pi
            Type       = Stationary state frequencies
            Prior      = Dirichlet

      3 --  Parameter  = Alpha
            Type       = Shape of scaled gamma distribution of site rates
            Prior      = Exponential(1.00)

      4 --  Parameter  = Pinvar
            Type       = Proportion of invariable sites
            Prior      = Uniform(0.00,1.00)

      5 --  Parameter  = Ratemultiplier
            Type       = Partition-specific rate multiplier
            Prior      = Fixed(1.0)

      6 --  Parameter  = Tau
            Type       = Topology
            Prior      = All topologies equally probable a priori
            Subparam.  = V

      7 --  Parameter  = V
            Type       = Branch lengths
            Prior      = Unconstrained:GammaDir(1.0,0.1000,1.0,1.0)



   The MCMC sampler will use the following moves:
      With prob.  Chain will use move
         0.93 %   Dirichlet(Revmat)
         0.93 %   Slider(Revmat)
         0.93 %   Dirichlet(Pi)
         0.93 %   Slider(Pi)
         1.85 %   Multiplier(Alpha)
         1.85 %   Slider(Pinvar)
         9.26 %   ExtSPR(Tau,V)
         9.26 %   ExtTBR(Tau,V)
         9.26 %   NNI(Tau,V)
         9.26 %   ParsSPR(Tau,V)
        37.04 %   Multiplier(V)
        12.96 %   Nodeslider(V)
         5.56 %   TLMultiplier(V)

   Division 1 has 413 unique site patterns
   Initializing conditional likelihoods
   Using standard SSE likelihood calculator for division 1 (single-precision)
   Initializing invariable-site conditional likelihoods

   Initial log likelihoods and log prior probs for run 1:
      Chain 1 -- -8643.988265 -- 42.620562
      Chain 2 -- -8383.035247 -- 42.620562
      Chain 3 -- -9054.857928 -- 42.620562
      Chain 4 -- -8757.217052 -- 42.620562

   Initial log likelihoods and log prior probs for run 2:
      Chain 1 -- -8752.756505 -- 42.620562
      Chain 2 -- -9161.226472 -- 42.620562
      Chain 3 -- -9070.270902 -- 42.620562
      Chain 4 -- -8759.113541 -- 42.620562

   Could not open file "primates.nex.mcmc"
   Error preparing print files
   Error in command "Mcmc"
```

**Comment by nylander** [comment]
I tried this now on a machine running 64-bit MS Windows 10 Enterprise but could not reproduce the error (downloaded and used binaries in https://github.com/NBISweden/MrBayes/releases/download/v3.2.6/MrBayes-3.2.6_WIN32_x64.zip).

I suspect that users may have an issue with write permissions in the directory where the output files are supposed to end up.

Closing this for now.
